deploy/windows/px-installer-node-win.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: px-installer-node-win
```

Comment:
Let's use a better name instead of calling it px-installer. Why not call it `px-csi-win-driver`?